test/CMakeLists.txt

```
@@ -371,13 +371,13 @@ else()
   set(_filter)
 endif()
 celeritas_add_test(celeritas/global/AlongStep.test.cc
-  NT 1 ${_optional_geant4_env} ${_filter}
+  NT 1 ${_optional_geant4_env} ${_needs_geo} ${_filter}
 )
 celeritas_add_test(celeritas/global/KernelContextException.test.cc NT 1
-  LINK_LIBRARIES ${_optional_json_link}
+  LINK_LIBRARIES ${_optional_json_link} ${_needs_geo}
```

```suggestion
   ${_needs_geo} LINK_LIBRARIES ${_optional_json_link}
```
Is a tiny bit less confusing (when not knowing the valid value for `_needs_geo` it could be a library).